### PR TITLE
Update the NEWS file for the new release

### PR DESCRIPTION
**Pull request by mattcaswell**

### CODE REVIEW

**Comment by mattcaswell** [comment]
Marked as urgent since this is needed for the release today

**Comment by t8m** [comment]
Is this the only major change?

What about db943f43 57225c99 e9cd6e7 - they should be at least mentioned in CHANGES.

**Comment by mattcaswell** [comment]
> Is this the only major change?

I didn't do a review of all the changes to see what updates should be made - on the assumption that we've updated CHANGES as we go (big assumption). I was more concerned with ensuring that NEWS was consistent. However I'll add some CHANGES entries for the items you pointed out.

**Comment by mattcaswell** [comment]
New commit pushed adding the above entries to CHANGES.

File: NEWS [hunk 1]
```diff
@@ -7,7 +7,8 @@
 
   Major changes between OpenSSL 1.1.1d and OpenSSL 1.1.1e [under development]
 
-      o
+      o Fixed an an overflow bug in the x64_64 Montgomery squaring procedure
```
Comment (InfoHunter) [inline NEWS]: There is an extra `an`

Comment (mattcaswell) [reply NEWS]: Oops. Fixed

**Comment by mattcaswell** [comment]
Ping

**Comment by mattcaswell** [comment]
Pushed. Thanks.

**Pull request closed without merging**